### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by renesass**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
-# 1.3.1
+# 1.3.2
+* Added stale bot support.
+* Checks on start if correct packages are installed.
```
Comment (eddiebergman) [inline changelog.md:3]: Where is this done?

**Review by eddiebergman — COMMENTED** [review]
Doesn't the module version also have to be bumped?

File: changelog.md (hunk 1 above)
Comment (renesass) [reply changelog.md:3]: Ah yeah, I removed it again.

File: smac/__init__.py [hunk 2]
```diff
@@ -19,4 +21,11 @@
     Matthias Feurer, André Biedenkapp, Difan Deng, Carolin Benjamins, Tim Ruhkopf, René Sass
     and Frank Hutter
 """
-version = "1.3.1"
+version = "1.3.2"
+
+
+if os.name != "posix":
+    print(
```
Comment (mfeurer) [inline smac/__init__.py:28]: How about `warning.warn`?

**Review by mfeurer — APPROVED** [review]

**Pull request merged**